boards/stm32f769i-disco/Makefile.dep

```diff
@@ -1,3 +1,7 @@
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+ifneq (,$(filter periph_fmc,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_fmc_32bit
```

Comment:
Why not use `periph_fmc_sdram` by default since the board provides it?

Reply:
It is realized in the same way as for the other boards according to your suggestion in https://github.com/RIOT-OS/RIOT/pull/19843#discussion_r1273015316.